### ISSUE TITLE
White screen problem of PI node software

### ISSUE THREAD

**Issue by Wu170** [pi-node/instructions#82]
Node software appears white screen problem for a long time, and it is difficult to solve it after exiting the software and reopening it. This is a annoying problem. I hope the team can fix this annoying problem
![屏幕截图 2021-03-06 201845](https://user-images.githubusercontent.com/71438288/110206729-81b3e780-7eba-11eb-967f-52eb2beb089f.png)

**Comment by ihatejam** [comment]
Instead of reopening the Node software to fix this, just click in the menu View -> Reload. 
I've also noticed this issue more when my earning cycle has stopped and I have not hit the lightning button in my mobile app yet.

**Comment by Wu170** [comment]
Yes, I tried the "reload" button, but it didn't seem to be so useful. It was a annoying problem, so I had to restart my computer.

**Comment by matejapoposki** [comment]
Is there any way around it?
- Uninstalled and re-installed Pi-Node
- menu View -> Reload

Blank screen. BUT the container in docker is still working fine and updating collecting data..
It worked for 4 months straight.  Now Blank screen after the update.

**Comment by jpshiomuni** [comment]
i am having same problem , after reinstalling my windows the PI NODE interface is blank . in the manu VIEW -reload doesnt brings any fruit . even my PI node interface is not being authenticated  with my mobile app because the pi node interface is totally blank. friend help me

**Comment by christiancdj89** [comment]
try to delete AppData\Roaming\Pi Network\Cache or rename the Folder "Cache" to "_Cache" 
restart the APP and it should Work, it worked for me every time :-)
